client/my-sites/email/titan-setup-thank-you/theme.ts

```diff
@@ -0,0 +1,37 @@
+export type ThankYouThemeProps = { theme?: ThankYouThemeType };
+export type ThankYouThemeType = {
+	colors: { textColorLight: string; borderColorLight: string; background: string };
+	breakpoints: {
+		desktopUp?: string;
+		tabletUp?: string;
+		bigPhoneUp?: string;
+		smallPhoneUp?: string;
+		tabletDown: string;
+	};
+	weights: {
+		bold: string;
+		normal: string;
+	};
+	fontSize: {
+		small: string;
+	};
+};
+
+const theme: ThankYouThemeType = {
+	colors: {
+		borderColorLight: 'var(--studio-gray-0)',
+		textColorLight: 'var(--color-text)',
+		background: 'white',
+	},
+	breakpoints: {
+		tabletDown: 'max-width: 782px',
+	},
+	weights: {
+		bold: '600',
+		normal: '400',
+	},
+	fontSize: {
+		small: '14px',
+	},
+};
+export default theme;
```

Comment:
I'm also not sure why we need that particular theme object: I believe you've introduced it to change the background color of the header but as far I can see in mockups and Figma the `Thank You` page for Yoast should look the same.

In a perfect world, we would move all common style settings under `client/components/thank-you` (so that it's easy to apply a change to all `Thank You` page) but I don't not how easy it is to do.

Reply:
I think there are two issues to address here:
1. Using the theme objects at all -- we have them, so we don't need to remove them yet.
2. Using _different themes_ for the two places we use the Thank You components.

I think we should definitely avoid (2) -- we're trying to make these designs look the same across the two use cases. Wherever possible we should be using the _same styles/config/theme_. So for this area, can we move these theme values into the Thank You component as much as possible?

Reply:
So, do you mean to add as component properties the needed values? Like colors?

Reply:
My thinking is that we currently have two separate `theme` objects defined, one in `client/my-sites/email/titan-setup-thank-you/theme.ts` and one in `client/my-sites/marketplace/theme.ts`, and we're passing both into the `ThankYou` component.

For now, I think we only want one theme defined in `client/components/thank-you/theme.ts` -- we want to have only one core theme for the Thank You component (at least for now) so we have a consistent experience. We can look at refactoring them out again later if we need more than one theme/style (e.g. for Jetpack thank you screens), but we want WordPress.com to start off consistently.

That said, I haven't looked closely at the elements for a few weeks, so it's possible that we do need two themes for now, but if that is the case, the two themes should _only_ pass in the things that need to be different, and should not replicate anything that should be the same and consistent across the two use cases.

Reply:
I managed to remove the theming. This makes simpler to call the component, as we don't have to provide a mandatory theme provider. Now, it will have 3 new more props: Background color for header, subtitle, and color for headings (title, subtite) in the header.

Seems to me simpler. Plus they are not mandatory and Yoast thank you page keeps working fine:
![image](https://user-images.githubusercontent.com/5689927/129369298-92529ba8-4609-45f9-b687-63ce5a40de47.png)